packages/smart-wallet/src/smartWallet.js

```diff
@@ -476,9 +476,8 @@ export const prepareSmartWallet = (baggage, shared) => {
               });
 
               const isOfferPossiblyExitable = !(
-                'error' in offerStatus || 'numWantsSatisfied' in offerStatus
```

Comment:
If there is an error, we don't necessarily want to forget about it just yet. The offer executor should automatically try to exit the offer in that case, and then `numWantsSatisfied` should come back as 0. However, the exit could fail if the rule is "after deadline" or something, so in that case keep it around so the user can manually try to exit it later.